packages/mobile/src/screens/sign-on-screen/components/SocialMediaSignUpButtons.tsx

```diff
@@ -30,11 +32,18 @@ export const SocialMediaSignUpButtons = ({
   page
 }: SocialMediaLoginOptionsProps) => {
   const { toast } = useToast()
-  const handleFailure = (e: unknown) => {
-    onError(e)
-    console.error(e)
-    toast({ content: socialMediaMessages.verificationError, type: 'error' })
-  }
+  const handleFailure =
+    (platform: 'twitter' | 'instagram' | 'tiktok') =>
+    (e: unknown, additionalInfo?: Record<any, any>) => {
+      onError(e)
+      reportToSentry({
+        level: ErrorLevel.Error,
+        error: e as Error,
```

Comment:
super nit: but maybe `e instanceof Error ? e : new Error(String(e))` in case it's not an `Error`?